static/app/components/dashboards/widgetQueriesForm.tsx

```diff
@@ -50,6 +50,8 @@ type Props = {
  * callback. This component's state should live in the parent.
  */
 class WidgetQueriesForm extends React.Component<Props> {
+  blurTimeout: ReturnType<typeof setTimeout> | undefined = undefined;
```

Comment:
Also I would probably just use `number`, since that's the return type of setTimeout.

Reply:
you don't need to get fancy with `ReturnType`

Reply:
Updated to `number`, also had to update `setTimeout` to `window.setTimeout` since I think ts is pulling the Node return type definition of `NodeJS.Timeout`